README.md

```diff
@@ -46,16 +46,21 @@ node validate.js
 
 #### Command usage:
 ```
-D:\sdk\openapi-validation-tools>node validate.js
-
-Usage: node validate.js <command> <spec-path> [--json]
-
-
+MacBook-Pro:openapi-validation-tools someUser$ node cli.js -h
 Commands:
-
-  - spec <raw-github-url OR local file-path to the swagger spec> [--json]    | Description: Performs semantic validation of the spec.
-
-  - example <raw-github-url OR local file-path to the swagger spec> [--json] | Description: Performs validation of x-ms-examples and examples present in the spec.
+  live-test <spec-path>         Performs validation of x-ms-examples and
+                                examples present in the spec.
+  validate-example <spec-path>  Performs validation of x-ms-examples and
+                                examples present in the spec.
+  validate-spec <spec-path>     Performs semantic validation of the spec.
+
+Options:
```

Comment:
log output to a file happens by default. may be we can ask the user if he/she wants to provide the file path for  storing the log file. We should also let them know the default log path.